### PR TITLE
Display information about udata's maintainance

### PR DESCRIPTION
**Pull request by bzg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ Customizable and skinnable social platform dedicated to (open)data.
 
 The [full documentation][readthedocs-url] is hosted on Read the Docs.
 
+udata is maintained by [Etalab](https://www.etalab.gouv.fr/), the
```
Comment (abulte) [inline README.md]: "mainly maintained" maybe?

Comment (bzg) [reply README.md]: I'd stick to "maintain". "Mainly maintains" is begin for questions on what "mainly" means.

"Maintains" clearly says that Etalab is responsible for stating the roadmap. It does not say that Etalab is the sole contributor nor that Etalab builds the roadmap in its corner. I will add a commit in this PR to explicit this, if you're okay.

Comment (abulte) [reply README.md]: I am!

**Comment by abulte** [comment]
@bzg please make a PR from your repo next time, as you can see Circle won't build a PR made on this repo.

**Comment by bzg** [comment]
Okay, point taken.

**Pull request merged**